assets/src/edit-story/components/panels/panel/shared/content.js

```diff
@@ -31,7 +31,7 @@ import { useContext } from 'use-context-selector';
 import panelContext from '../context';
 
 const Container = styled.div`
-  padding: ${({ padding }) => padding || '4px 16px 8px'};
+  padding: ${({ padding }) => padding || '8px 16px 8px'};
```

Comment:
Adjusted this since the Video Settings panel needed more room for not cutting off the Checkbox animation.